### PR TITLE
Update neighborhood data

### LINKED ISSUE #562: Filter out and remove S3 .gitkeep files
At some point, probably in deployment, `.gitkeep` files are being added along with the neighborhood images to the **/assets/neighborhoods/** folder of the production and staging site buckets in S3. This issue includes finding where that addition is occurring and changing it so `.gitkeep` does not get pushed up, and removing it from where it already exists in those buckets.

### LINKED ISSUE #563: Update neighborhood data

### PR DESCRIPTION
**Pull request by aaronxsu**
## Overview

This PR:
- adds the updated `neighborhood.csv`
- updates the `neighborhood_data/update_data.sh` script to remove unnecessary steps
- updates the `scripts/imagepublish` script to not upload `.gitkeep` to S3

Some things not visible in the code change:
1. `neighborhood.json` and `neighborhood_bounds.json`, which are results of the `neighborhood_data/update_data.sh` script were pushed to the data bucket on S3
2. the updated images are pushed to the staging site bucket under `assets/neighborhoods`

### Checklist

- [X] Run `./scripts/format` to lint, format, and fix the application source code.

### Demo

This was the output from running the management script in my local dev database. The neighborhood records are updated:

```
$ AWS_PROFILE=echo-locator ./scripts/manage import_neighborhood_data \
>     s3://echolocator-production-data-us-east-1/neighborhoods.json \
>     s3://echolocator-production-data-us-east-1/neighborhood_bounds.json \
>     s3://echo-locator-staging-site-us-east-1/assets/neighborhoods/
Starting echo-locator_database_1 ... done
Creating echo-locator_django_run ... done
Done!

0 neighborhoods created, 236 updated
```

### Notes

1. The shape of the updated `neighborhood.csv` is consistent with the previous version. Thus, no script change needed
2. I plan to perform the following if the changes here look good and works well in the reviewer's local dev DB:
    - upload the images to the production site bucket under `assets/neighborhoods`
    - run the management script `import_neighborhood_data` on staging
    - add the corresponding steps to https://github.com/azavea/echo-locator/issues/502

## Testing Instructions

 * Pull down this branch, and run `./update_data.sh` in `neighborhood_data` directory. Make sure the neighborhood related JSON and images are generated correctly without terminal errors
 * Run `./scripts/imagepublish staging` and make sure the images are pushed to the staging site bucket under `assets/neighborhoods` prefix correctly and without the `.gitkeep` file
 * Run the following management script to update the neighborhood data in your local dev DB.
```
AWS_PROFILE=echo-locator ./scripts/manage import_neighborhood_data \
>     s3://echolocator-production-data-us-east-1/neighborhoods.json \
>     s3://echolocator-production-data-us-east-1/neighborhood_bounds.json \
>     s3://echo-locator-staging-site-us-east-1/assets/neighborhoods/
```
 * Spin up the app and browse some neighborhood info on the frontend, spot check some and make sure they still show up fine

Resolves https://github.com/azavea/echo-locator/issues/563
Resolves https://github.com/azavea/echo-locator/issues/562

### CODE REVIEW

File: scripts/imagepublish [hunk 1]
```diff
@@ -25,6 +25,6 @@ then
             exit 1
     else
         TAUI_SITE_BUCKET="s3://echo-locator-${1}-site-us-east-1"
-        aws s3 cp --profile echo-locator --recursive neighborhood_data/images/ "${TAUI_SITE_BUCKET}/assets/neighborhoods/"
+        aws s3 cp --profile echo-locator --recursive --exclude "*.gitkeep" neighborhood_data/images/ "${TAUI_SITE_BUCKET}/assets/neighborhoods/"
```
Comment (ddohler) [inline scripts/imagepublish:28]: Thanks for taking care of this!

File: src/backend/api/management/commands/import_neighborhood_data.py [hunk 2]
```diff
@@ -126,8 +126,6 @@ def handle(self, *args, **options):
         neighborhood_images = [
             Path(img.key).name
             for img in image_bucket.objects.filter(Prefix=images_url_parts.path[1:])
-            # Remove following completion of issue 562 (https://github.com/azavea/echo-locator/issues/562)
```
Comment (ddohler) [inline src/backend/api/management/commands/import_neighborhood_data.py:129]: GitHub issue links in comments FTW! 🎉

**Review by ddohler — APPROVED** [review]
Looks great! I spot-checked a few neighborhoods and confirmed that the rents all match the new file (and not the old file).

File: scripts/imagepublish (hunk 1 above)
Comment (aaronxsu) [reply scripts/imagepublish:28]: No problem!

File: src/backend/api/management/commands/import_neighborhood_data.py (hunk 2 above)
Comment (aaronxsu) [reply src/backend/api/management/commands/import_neighborhood_data.py:129]: Indeed! So glad that we have a comment there.

**Pull request merged**